graph/types/media.graphqls

```diff
@@ -0,0 +1,17 @@
+# Maybe make a basePost interface and extend
+# Media can also just be data on a post
+type Media {
+    id: ID!
+    createdAt: String!
+    isDeleted: Boolean!
+    deletedReasonCode: PostDeletedReason
+    mediaType: String!
```

Comment:
So if this is deleted, then mediaType, mediaSize, and assetLocation will all be `null`. I'm just realizing we may have this problem in other types as well. let's clean it up here first though.